WooCommerce/Classes/ViewRelated/OrdersBadgeController.swift

```diff
@@ -34,25 +34,36 @@ private extension OrdersBadgeController {
     enum Constants {
         static let diameter    = CGFloat(10)
         static let borderWidth = CGFloat(1)
-        static let xOffset     = CGFloat(10)
-        static let yOffset     = CGFloat(-6)
+        static let xOffset     = CGFloat(14)
+        static let yOffset     = CGFloat(1)
         static let tagOffset   = 999
-        static let width       = CGFloat(17)
-        static let height      = CGFloat(17)
+        static let width       = CGFloat(15)
+        static let extendWidth = CGFloat(23)
+        static let height      = CGFloat(15)
     }
 
     func badge(for tab: WooTab, with text: String) -> BadgeLabel {
+        var width = Constants.width
+        var horizontalPadding = CGFloat(2)
+
+        let ninePlus = NSLocalizedString("9+", comment: "A badge with the number of orders. More than nine new orders.")
+        if text == ninePlus {
```

Comment:
would it be more flexible if this check is on the text length, like `text.count > 1`?

Reply:
Maybe. I try not to assuming things based on English (for example, a language may just stop counting and say "ten", and it may be a single character to represent "ten"), and I'm not sure what it looks like for non-roman numeral systems.